apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -142,6 +142,10 @@ export const cli = {
   items: [
     { name: 'Overview', url: '/guides/cli' },
     { name: 'Managing Environments', url: '/guides/cli/managing-environments' },
+    {
+      name: 'Using environment variables in your config',
+      url: '/guides/cli/using-envitonment-variables-in-your-config',
```

Comment:
```suggestion
      name: 'Using environment variables in config.toml',
      url: '/guides/cli/using-envitonment-variables-in-config',
```

I suggest we avoid words like "your" in page titles